carma_wm/include/carma_wm/CARMAWorldModel.h

```diff
@@ -176,7 +180,7 @@ class CARMAWorldModel : public WorldModel
   LaneletRoutingGraphConstPtr getMapRoutingGraph() const override;
 
   lanelet::Optional<TrafficRulesConstPtr>
-  getTrafficRules(const std::string& participant = lanelet::Participants::Vehicle) const override;
+  getTrafficRules(const std::string& participant) const override;//Participant type is sent from WMListenerWorker
```

Comment:
I think you ought to have two versions of this method. 
This one where the user can provide the participant type in question and therefore not use the WMListener set value.
The other should be a zero argument method which uses the default WMListener set value in the world model. 
getTrafficRules()

That will maintain API compatibility with the existing code that relied on the default parameter.